experimental/fairseq_moe_docker_slurm/readme.md

```diff
@@ -15,30 +15,34 @@ Total number of paramters = Number of GPUS * 2B + 4.5B
 - Compute node(s), ND96asr_v4 (Running Ubuntu-hpc 18.04)
 - Azure container registry deployed
 
-## Docker set-up on compute nodes and scheduler
+## Enroot authentication
 
 ```
-pdsh -w ^/path/to/hostfile sudo </path/to/docker_setup.sh
+mkdir -p $HOME/.config/enroot/
+echo "machine nvcr.io login $oauthtoken password YOUR_KEY" > $HOME/.config/enroot/.credentials
 ```
->Note: Make sure scheduler and the compute nodes have the name GID for the docker group. Modify script update "USER".
+>Note: You need to get your key from [Nvidia API Key Page](https://ngc.nvidia.com/setup/api-key)
 
-
-## Build docker container
+## Build enroot squashfs image
 
 ```
-az acr build --registry <your_acr_name> --image docker/<container_name:version> .
+enroot import docker://nvcr.io/nvidia/pytorch:21.10-py3
+enroot create --name pytorch nvidia+pytorch+21.10-py3.sqsh
+enroot start --root --rw --mount .:/workspace pytorch
+```
+Exit the container and export the updated version
+```bash
```

Comment:
Are you missing a step to update the container?

Reply:
Could you also have used the container in ACR to run this benchmark instead of creating a new container?

Reply:
The enroot export step will create a new squashfs image which will be used to run the benchmark. 

ACR could be used. But I think this workflow is more lightweight since users don't have to deal with ACR. Once the initial image is imported with enroot everything stays local.